docs/openshift.asciidoc

```diff
@@ -5,16 +5,18 @@ This section explains how to run ECK on OpenShift.
 
 NOTE: Only Elasticsearch and Kibana are supported on OpenShift.
 
+[float]
 === Prerequisites
 
-. To run the following instructions, you must be a `system:admin` user or a User with the privileges to create Projects, CRDs, and RBAC resources at the cluster level.
+. To run the following instructions, you must be a `system:admin` user or a user with the privileges to create Projects, CRDs, and RBAC resources at the cluster level.
 
 . Set virtual memory settings on the nodes
 +
-Before deploying an Elasticsearch cluster with ECK, make sure you correctly applied the `vm.max_map_count` setting on all the nodes of your cluster. Pods created by ECK are likely to run with the `restricted` https://docs.openshift.com/container-platform/4.1/authentication/managing-security-context-constraints.html[Secure Context Constraints] (SCC), they will run with a limited set of privileges and will not be able to change this setting on the nodes that host them. You can refer to the following https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html[documentation] for more information.
+Before deploying an Elasticsearch cluster with ECK, make sure you correctly applied the `vm.max_map_count` setting on all the nodes of your cluster. Pods created by ECK are likely to run with the `restricted` https://docs.openshift.com/container-platform/4.1/authentication/managing-security-context-constraints.html[Secure Context Constraints] (SCC), they run with a limited set of privileges and cannot change this setting on the nodes that host them. For more details, see the https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html[Virtual memory].
```

Comment:
Should the link at the end somehow make clear that it is not part of the ECK docs but Elasticsearch documenation?

Reply:
Would that be OK?

> For more details, see the Elasticsearch documentation on https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html[Virtual memory].

Reply:
I like it 👍